src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -402,6 +402,10 @@ public List<OffenderIdentifier> getOffenderIdentifiers(final Long bookingId, @Nu
             .toList();
     }
 
+    public List<Alias> getAliases(final Long bookingId) {
+        return repository.findInmateAliasesByBooking(bookingId, "createDate", Order.ASC, 0, 100).getItems().stream().toList();
+    }
```

Comment:
Put this in front of the repository so we don't get tied to the repo in PrisonerSearchService.